pybot/endpoints/slack/message_templates/commands.py

```diff
@@ -29,7 +29,7 @@ def mentor_request_attachments(services, mentors, skillsets):
                     "text": "Service",
                     "name": "service",
                     "options": [
-                        {"text": service, "value": service} for service in services
+                        {"text": service, "value": service} for service in services.sort()
```

Comment:
You'll want to change each of these to `sorted(services)`.  `list.sort` sorts the list in place and returns `None`, `sorted() creates a new list, sorts it, and returns it.